README.md

```diff
@@ -0,0 +1,91 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
```

Comment:
- [OPTIONAL] I can see you are using the Microverse template, good job 👏, but I think you forgot to add the project name. I suggest adding it.